docs/source/datasources/glofas.rst

```diff
@@ -0,0 +1,316 @@
+#. Load the data. The processed files, which are split by time in the same
+   way as the raw data,
+   are read in as a single `xarray.DataSet`.
+
+
+The download step makes use of the
+`CDS API
+<https://cds.climate.copernicus.eu/api-how-to>`_.
+You will need to register an account on CDS, then once logged in go to your
+`user page
+<https://cds.climate.copernicus.eu/user/>`_
+and note down your UID and API key.
+
+The
+`how-to page
+<https://cds.climate.copernicus.eu/api-how-to>`_
+has instructions for each operating system, but we summarize the instructions below:
+
+Windows: Create a file called `%USERPROFILE%\.cdsapirc` where, `%USERPROFILE%` is usually located
+in your `C:\Users\Username` folder.
+
+Mac and Linux: Create the file `~/.cdsapirc`.
+
+Then, for all operating systems, add the following to the created file:
+
+.. code-block:: shell
+
+    url: https://cds.climate.copernicus.eu/api/v2
+    key: $UID:$API-key
+
+where `$UID` is your UID and `$API-key` is your API key.
+
+Next, if it :ref:`doesn't already exist<list of supported countries>`,
+you need to create a country configuration
+for the country you would like to analyze.
+
+An example country config for Bangladesh is:
+
+.. code-block:: yaml
+
+    iso3: bgd
+    glofas:
+      reporting_points:
+      - name: Bahadurabad
+        lon: 89.65
+        lat: 25.15
+      - name: Hardinge Bridge
+        lon: 89.05
+        lat: 24.05
+
+The reporting points indicate the raster file coordinates used
+to extract the river discharge for a particular location.
+Those in the above example have been taken from the
+`GloFAS map viewer
+<https://www.globalfloods.eu/glofas-forecasting/>`_.
+If you select "Reporting Points" from the "Hydrological"
+menu at the top, they will appear as dots on the map. If you then
+click on one of the points, you are able to see
+information such as the station name, and LISFLOOD X and Y, which are
+the respective longitude and latitude used in the configuration file.
+
+Reporting point coordinates are manually selected by the GloFAS team to
+be representative of physical gauge locations, and to be located on a river
+in the model raster file. In principle, one could
+specify any set of coordinates that exists on the raster, but caution is advised
+when doing so.
+
+You can initialize a built-in country config as follows:
+
+.. code-block:: python
+
+    from aatoolbox import create_country_config
+
+    country_config = create_country_config(iso3="bgd")
+
+Another required input is the geographic area of interest, which will
+define the bounds of raw raster data to be downloaded. A simple
+way to identify the area around the chosen country is to use the COD
+administrative boundaries. You will need to download the data,
+and extract a geo bounding box:
+
+.. code-block:: python
+
+    from aatoolbox import CodAB, GeoBoundingBox
+
+    codab = CodAB(country_config=country_config)
+    codab.download()
+    admin0 = codab.load()
```

Comment:
get an error here if I don't include `admin_level=0` argument

Reply:
Can you post the error traceback? I won't change the documentation because the error means there is a bug in COD AB which should be fixed